packages/qwik-router/src/runtime/src/head.ts

```diff
@@ -37,9 +37,10 @@ export const resolveHead = (
     }
     return data;
   }) as any as ResolveSyncValue;
+  const storeEv = (globalThis as any).qcAsyncRequestStore?.getStore?.();
```

Comment:
You're not actually using storeEv, so just the existence of qcAsyncRequestStore is enough to know that Core has one too